R/lazyframe__lazy.R

```diff
@@ -1952,3 +1952,60 @@ LazyFrame_group_by_dynamic = function(
   ) |>
     unwrap("in $group_by_dynamic():")
 }
+
+#' Plot the query plan
+#'
+#' Note that this requires to have the package `DiagrammeR` installed. Use
+#' `raw_output = TRUE` to obtain the raw GraphViz code without plotting it.
+#'
+#' @param optimized Optimize the query plan.
+#' @param raw_output Return the raw GraphViz code only, does not print the plot.
+#' @inheritParams LazyFrame_set_optimization_toggle
+#' @param ... Additional arguments passed to `DiagrammeR::grViz()`.
+#'
+#' @return If `raw_output = TRUE`, a character vector. Otherwise, an object of
+#' class `"grViz"`.
+#'
+#' @examplesIf requireNamespace("DiagrammeR", quietly = TRUE)
+#' query = pl$LazyFrame(mtcars)$
+#'   filter(pl$col("drat") > 3)$
+#'   with_columns(foo = pl$col("mpg") + pl$col("cyl"), bar = pl$mean("mpg"))
+#'
+#' query$show_graph(raw_output = TRUE)
+#'
+#' query$show_graph()
+LazyFrame_show_graph = function(
+    optimized = TRUE,
+    raw_output = FALSE,
+    type_coercion = TRUE,
+    predicate_pushdown = TRUE,
+    projection_pushdown = TRUE,
+    simplify_expression = TRUE,
+    slice_pushdown = TRUE,
+    comm_subplan_elim = TRUE,
+    comm_subexpr_elim = TRUE,
+    streaming = FALSE,
+    ...) {
```

Comment:
Is there a reason for the order of the arguments?
It seems that positional arguments are not allowed in Python.

Reply:
That was just a mistake, fixed now